src/ocamlorg_frontend/pages/packages.eml

```diff
@@ -6,7 +6,15 @@ Layout.render
 ~description:"Discover thousands of community packages and browse their documentation."
 ~canonical:Url.packages
 ~active_top_nav_item:Header.Packages @@
-<div class="intro-section-simple">
+
+let search_dropdown () =
+  <div id="packages-search-results" aria-live="polite" ></div>
+  <a class="flex py-2 px-4 gap-4 hover:bg-primary-100 font-normal rounded-md text-primary-600" href="<%s Url.api %>">
+    Standard Library API
+    <%s! Icons.arrow_top_right_on_square "w-6 h-6"%>
+  </a>
+in
```

Comment:
seems to be the same function as in `header.eml`, should be factored out to a place where it can be shared.

`packages_autocomplete_fragment.eml` seems to be a reasonable place.